src/pages/en/reference/cli-reference.md

```diff
@@ -48,15 +113,15 @@ This command only checks types within `.astro` files.
 
 📚 Read more about [TypeScript support in Astro](/en/guides/typescript/).
 
-### `astro add`
+## `astro add`
 
-Adds an integration to your configuration.
+Adds an integration to your configuration. Read more in [the integrations guide](/en/guides/integrations-guide/#automatic-integration-setup).
 
-### `astro docs`
+## `astro docs`
 
 Launches the Astro Docs website directly from the terminal.
 
-### `astro telemetry`
+## `astro telemetry`
 
 Sets telemetry configuration for the current user. Telemetry is anonymous data that provides insights into which features are most often used.
```

Comment:
Taking the opportunity to try and make it clear that Astro telemetry monitors _you_ as the CLI user, not your end user, and that the insights are for "us" (Astro) not something you can view
```suggestion
Sets telemetry configuration for the current CLI user. Telemetry is anonymous data that provides us insights into which Astro features are most often used.
```

Reply:
Agree on this. Do you think “Astro team” rather than “us” is clearer?

```suggestion
Sets telemetry configuration for the current CLI user. Telemetry is anonymous data that provides the Astro team insights into which Astro features are most often used.
```